### PR TITLE
[Proposal] change underscore to lodash

### PR DESCRIPTION
**Pull request by sugarshin**
Thank you merged prev PR !

`lodash` has `underscore` compatible, it will also be small fill size after build by require for each function.
This, I think that problem is solved in the case of dependent on the large library, such as the `underscore`.

### CODE REVIEW

**Comment by tleunen** [comment]
Can we please remove all lodash functions except throttle?
We can use all native functions instead.

**Comment by sugarshin** [comment]
pls check it out.

File: src/modifiers.jsx [hunk 1]
```diff
@@ -25,7 +24,8 @@ var Modifiers = React.createClass({
     var list = [];
     var onChange = this.props.onChange;
     
-    _.each(this.props.modifiers, function(hex, index){
+    Object.keys(this.props.modifiers).forEach(function(index){
+      var hex = this.props.modifiers[index];
```
Comment (tleunen) [inline src/modifiers.jsx]: You should pass the context `this` to `forEach` if you want to use it. Or keep a variable outside of the function to access the modifiers.

**Comment by tleunen** [comment]
In general, I think it looks ok, but please remove all `.bind()` in render and I'll review once again after :)

**Comment by sugarshin** [comment]
How's this?

```js
Object.keys(this.props.modifiers).forEach((function(_this){
  return function(index){
    var hex = _this.props.modifiers[index];
    ...
  };
})(this));
```

**Comment by tleunen** [comment]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

More like this:
```js
Object.keys(this.props.modifiers).forEach(function(index) {
    var hex = this.props.modifiers[index];
}, this);
// or
var modifiers = this.props.modifiers;
Object.keys(modifiers).forEach(function(index) {
    var hex = modifiers[index];
});
```

**Comment by sugarshin** [comment]
How about?

**Comment by tleunen** [comment]
looks good

**Comment by sugarshin** [comment]
@tommoor pls review :)

**Comment by tommoor** [comment]
Hey both, just had a quick skim and this looks good - I'm going to do my best to test and merge with a new release in next 24hours, bear with me :)

**Comment by tommoor** [comment]
Thanks for your work here! I implemented this in 457f5aad7a6a3fcfed7d8194da96a448b010e2f5 as the extra code for native methods is not worth the loss of readability in my mind.

**Pull request closed without merging**